HelpScout/ruleset.xml

```diff
@@ -46,11 +46,6 @@
  <!-- Add additional Help Scout sniffs -->
  <rule ref="HelpScout.Functions.DisallowXDebug"/>
 
- <!-- Allow us to use the compact phpdoc format -->
- <rule ref="Generic.Commenting.DocComment">
-  <exclude name="Generic.Commenting.DocComment.MissingShort"/>
```

Comment:
I'm confused by this change. This rule on line 51 specifically `excludes` the Missing Short rule. Won't removing this block enable it again?

Reply:
It confused me too, but when I run PHPCS on the file with the short dockblock I get the following errors:

```
FILE: /hs-app/Test.php
----------------------------------------------------------------------
FOUND 2 ERRORS AFFECTING 1 LINE
----------------------------------------------------------------------
 7 | ERROR | [x] The open comment tag must be the only content on the
   |       |     line
 7 | ERROR | [x] The close comment tag must be the only content on
   |       |     the line
----------------------------------------------------------------------
PHPCBF CAN FIX THE 2 MARKED SNIFF VIOLATIONS AUTOMATICALLY
----------------------------------------------------------------------
```

Using PHPCBF to fix it results in it messing up the indentation.

After removing the rule (as in this PR) then there are no PHPCS errors.

It seems backwards, but that is what I'm seeing happening.

Reply:
Although I've only done an isolated test so far, so I'll do some more testing with real code to see if removing the rule causes some other problems.

Reply:
If it works. Go for it.